src/checkers/periodicConfigMapChecker.go

```diff
@@ -149,10 +158,17 @@ func (p *PeriodicConfigMapChecker) StartChecking() {
 
 					password, err := getPasswordFromSecret(client, configMap.Namespace, configMap.Name+"-password", strings.Split(name, ".")[0]+".password")
```

Comment:
What case does `strings.Split(name, ".")[0]+".password"` handle? I thought the full secret name would always be the key.

Reply:
If jit generated secret is 
{
ess.ca: ...
ess.key: ...
ess.p12: ...
ess.jks
ess.crt ...
}, then the corresponding password secret is 
{
ess.password: ... 
}
strings.Split(name, ".")[0]+".password" is to get ess.password from ess.p12.

Reply:
Thanks, what would the value of `name` be in the second case then? `getPasswordFromSecret(client, configMap.Namespace, configMap.Name+"-password", name+".password")`

Reply:
This one I created for password protected non jit certs. https://teletracking.atlassian.net/wiki/spaces/DEPL/pages/3507061128/Client+Cert+Monitoring+-+How-To
Just figured it would be cleaner if we ask people to follow this convention if they want monitoring for non jit cert. jit creates key.p12 and key.password but for a non jit cert, we can't assume there would periods in name like people can just use key_p12, so we can ask them to just use key_p12.password as the key in the password secret.